### ISSUE TITLE
Define website MVP features 🚀

### ISSUE THREAD

**Issue by mmahdy-modernise** [modernise-io/.github#1]
These are the features I expect from modernise website:

- Home page that defines our statement
- Home page that contains media (Videos & Pictures)
- Portfolio page (Should we copy single click?)
- Contact us page
- Tech Radar
- Technical blog (For marketing purposes)
- Careers page
- Services page (To explain the services we offer)
- WOW page (To explain how we work and why we differ than the market)
- The website should have impressive UX & UI to attract customers (Websites show your quality)

**Comment by mohamed-modernise** [comment]
Yep. Agreed, some were in fact identified on a Miro board. That's shared over Slack
let's have a backlog of the features, with some could be ready in the near future and some are thought leadership for next year.
